docs/README.md

```diff
@@ -24,24 +24,70 @@ This repository is where the declaration of remote doctypes is done. Read more a
 
 - [Bets Unibet](com.unibet.bets): Bets from Unibet website
 
-## Metadata
+## Generic model
 
-Every doctype should have `metadata` fields.
+### Relationships
 
-- `version` is be useful for migrations
-- `dateImport` is useful for debugging purposes
+Relations between documents are under a `relationships` object at the root of the document. Relations are referenced by their names.
+
+Each relation is an object with a `data` property containing `null`, one reference or an array of references.
```

Comment:
Nitpick, but without `either` you'd think it contains `null` AND `one reference` because you haven't read the `or` word yet.

```suggestion
Each relation is an object with a `data` property containing either `null`, one reference or an array of references.
```